node/src/service.rs

```diff
@@ -766,6 +771,192 @@ where
 	Ok((task_manager, client))
 }
 
+/// Start a node with the given parachain `Configuration` and relay chain `Configuration`.
```

Comment:
@RustNinja where is all of this code coming from? Is the code duplication now even getting worse?

If it is just duplicated code, then fine. If there are any crucial changes to the duplicated code here, can you point them out?